src/os/bluestore/BlueStore.h

```diff
@@ -945,6 +961,7 @@ class BlueStore : public ObjectStore,
     bluestore_onode_t onode;  ///< metadata stored as value in kv store
     bool exists;              ///< true if object logically exists
 
+    boost::object_pool<Extent> extent_pool{4096,4096};
```

Comment:
If I understand these parameters correctly - IMO they look too high for many use patterns. This is especially valid for the first parameter - just a single freed onode will use 4K * 8 bytes, wouldn't it?